docs/instrumentation/tracing-java-interop.md

```diff
@@ -0,0 +1,208 @@
+# Tracing - interop with Java-instrumented libraries
+
+## Glossary
+
+| Name                                     | Description                                                  |
+|------------------------------------------|--------------------------------------------------------------|
+| [Context][otel4s-context]                | otel4s context that carries tracing information (spans, etc) |
+| [Local{F, Context}][cats-mtl-local]      | The context carrier tool within the effect environment       |
```

Comment:
I think escaping `[``]` would be better than `{``}` for `Local`, if possible. I don't recall the syntax though >.>

Reply:
Hm, I tried to escape `[` and `]` but it didn't work :( I will double check

Reply:
I ended up using similar Unicode characters:
https://www.compart.com/en/unicode/U+FF3B
https://www.compart.com/en/unicode/U+FF3D